src/Main.tsx

```diff
@@ -55,6 +55,10 @@ export type MainStackParamList = {
     secretId: string
     token: string
     uniqueId: string
+    packageInfo?: {
+      name?: string
+      version?: string
+    }
```

Comment:
shall we create an issue to track making this (and changes done in other projects) mandatory some time in the future?

Reply:
I would say so, but is there a timeframe where it would be fine to change that? Because it would break users workflow until they update the action and since we don't receive any update notifications for action releases (and dependabot won't work with actions either) they will never know the action changed.

Reply:
Actually I think we can change our current version of the action and add a warning telling the user to update until a certain date (a couple of months from now probably since this feature is not really important) when we would remove API's backwards compatibility. That way I think most people would notice and update the action since the user won't have to change anything on their side.

When we do go forward with the API update we should also add a helpful error message for the user explaining why their CI is broken and what to do to fix it.

Reply:
A better way to handle this is to release a major of the action instead, users don't look at warnings

Reply:
but this won't really solve the issue I don't think. The users will have no way of knowing that we released a new major since they won't be notified and everyone on the old version will have the CI broken when we finally remove backwards compatibility